taxonomies/categories.txt

```diff
@@ -1562,11 +1809,13 @@ origins:en: fr:Bretagne
 wikidata:en:Q2905049
 
 <en:Beers
-en:Beers from Ireland, Irish beers
+<en:Country specific beers
```

Comment:
Should we keep Irish beers directly under Beers, or have it under Country specific beer, which is itself under Beers? If we want the latter, then we need to remove the <en:Beers from Irisih beers etc.

Reply:
Took me a while to understand. But removed it in [699ec3e](https://github.com/openfoodfacts/openfoodfacts-server/pull/8718/commits/699ec3e7539ad8f6c7d7c17b690005df18620b82)